platform-enterprise_docs/studios/add-studio-git-repo.md

```diff
@@ -0,0 +1,132 @@
+- Git repositories with multiple Studio configurations are not supported. However, it is possible to use a Git repository with multiple branches and a single configuration per branch.
+
+### Create the required configuration files 
+
+**.seqera/studio-config.yaml configuration file**
+
+Create a `.seqera/studio-config.yaml` file in the the `.seqera/ ` directory in your repo. The only required field is `session.template.kind`. All other fields are optional.
+
+
+```yaml
+schemaVersion: "0.0.1"
+kind: "studio-config"
+session:
+    name: "studio-name"                          # Must be unique to a workspace. If undefined, an auto-generated name is used
+    description: "desc"                          # Short description of what the Studio is for
+    template:
+        kind: "registry"|"dockerfile"|"none"     # Required
+        registry: "cr.seqera.io/image:latest"    # Ignored for `dockerfile` and `none`
+        dockerfile: "Dockerfile"                 # Ignored for `registry` and `none`
+    clone:
+        enabled: true                            # Clone the contents of the repository to the Studio. Defaults to `true`
+        path: "/workspace"                       # Defaults to `/workspace`. If you want to clone to `/workspace/repository` then you need to specify this
+    dependencies:
+        condaEnvironmentFile: "environment.yaml" # Define additional libraries (and versions). Ignored for `dockerfile`
+    computeRequirements:
+        awsBatch:                                # Ignored for non-AWS batch CE
+            cpu: 2                               # Number of CPUs to use. Defaults to 2
+            gpu: 0                               # Number of GPUs to use (if the CE supports GPUs). Defaults to 0
+            memory: 8192                         # Memory allocated in MiB. Defaults to 8192.
+    environmentVariables:                        # Ordered sequence of elements that are objects (or mappings) of key-value pairs
+        -   name: "var1"
+            value: "value1"
+        -   name: "var2"
+            value: "value2"
+    management:                                  # Session management settings
+        lifespanHours: 1                         # Ignored if workspace lifespan is set
+        isPrivate: false                         # Defaults to `false`
+```
+
+The schema can define a Dockerfile, which has to be inside the `.seqera` folder. The following limitations apply:
+
+- You need to configure a target repository using the `TOWER_DATA_STUDIO_WAVE_CUSTOM_IMAGE_REGISTRY` and `TOWER_DATA_STUDIO_WAVE_CUSTOM_IMAGE_REPOSITORY` environment variables. If no repository configuration is specified, the build will fail.
+- Each workspace needs to have credentials available in the workspace to push to the repository you've specified.
+- The only supported repository and compute environment combination for a fully private Dockerfile-based Studio is ECR and AWS.
```

Comment:
Feels like this should be the first listed limitation as it's an infrastructure requirement